Context/ConfiguredContext.php

```diff
@@ -60,6 +60,9 @@ class ConfiguredContext implements SearchContextInterface
      */
     private $interceptorProvider;
 
+    /** @var ContextSortAttribute */
```

Comment:
It doesn't seem appropriate to add this as an instance variable - it's a function (which isn't going to be dynamically modified), not data.  I would just keep everything as is but use the new class within the existing private method - seems to me to retain appropriate levels of abstraction in the right places.